localPackages/site_developer/Resources/Private/Layouts/Page/Default.html

```diff
@@ -0,0 +1,27 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+    data-namespace-typo3-fluid="true">
```

Comment:
Would this also fit in one line? (Ditto in the other files with the same snippet.)